.github/workflows/linters.yml

```diff
@@ -0,0 +1,70 @@
+name: Linters
+
+on: pull_request
+
```

Comment:
- [ ] Your readme file looks like a template :smile: because it doesn't have any information regarding you and your application :thinking: Kindly insert your application information in the respective sections to make your readme file descriptive and professional :blush:

Hint: You can see this [readme](https://github.com/samlorlah/recipe-app/blob/dev/README.md) file to get an idea :+1: